ts/components/wallet/paymentSummary/PaymentSummaryComponent.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * this component will display the transaction details if no updates on the amount are not identified
+ */
+import { H1, H3, Right, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import I18n from "../../../i18n";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+  amount: string;
+  expireDate: string;
+  tranche: string;
+}>;
+
+const styles = StyleSheet.create({
+  contentPadding: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  },
+
+  darkContentPadding: {
+    backgroundColor: variables.brandDarkGray,
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  },
+
+  white: {
+    color: variables.brandLight
+  },
+
+  whiteStrike: {
+    fontSize: variables.fontSizeBase * 1.25,
```

Comment:
`variables` already contains different font sizes based on `fontSizeBase`: 1.25x is missing, but please consider whether the available font sizes are suitable anyway. If not, it might be worth considering adding this new derived font size as part of `variables` if it is recurrent throughout the code